pygdbmi/tests/test_app.py

```diff
@@ -131,6 +133,66 @@ def test_controller_buffer(self):
         assert(r['type'] == 'result')
         assert(r['payload'] == {'BreakpointTable': {'nr_cols': '6', 'nr_rows': '1'}})
 
+        """
```

Comment:
the triple quote should only be used for docstrings (https://stackoverflow.com/questions/7696924/way-to-create-multiline-comments-in-python#7696966)